README.md

```diff
@@ -314,20 +324,34 @@ Openstack Object Storage settings
 
 This plugin requires various trackers to be backported to maintain the plugin functionality.
 
-| Moodle version   | Totara version     | Branch       | PHP  |Mandatory patches | Pre-signed URLs |
-|------------------|--------------------|--------------|------|------------------|-----------------|
-| Moodle 3.9       |                    | master       | 7.0+ |                  |                 |
-| Moodle 3.8       |                    | master       | 7.0+ | MDL-58281         | MDL-68342       |
-| Moodle 3.4 - 3.7 |                    | master       | 7.0+ | MDL-58281         | MDL-68342, MDL-66304 |
-| Moodle 3.3       | Totara 12          | master       | 7.0+ | MDL-58281         | MDL-68342, MDL-53240,<br>MDL-66304 |
-| Moodle 2.9 - 3.2 | Totara 2.9, 9 - 11 | 27-32-STABLE | 5.5+ | MDL-58281, MDL-46375,<br>MDL-58068, MDL-58684,<br>MDL-58297 | MDL-68342, MDL-53240,<br>MDL-66304 |
-| Moodle 2.7 - 2.8 | Totara 2.7 - 2.8   | 27-32-STABLE | 5.5+ | MDL-58281, MDL-46375,<br>MDL-58068, MDL-49627,<br>MDL-58684, MDL-58297 | MDL-68342, MDL-53240,<br>MDL-66304 |
+| Moodle version   | Totara version     | Branch       | Mandatory patches | Pre-signed URLs |
```

Comment:
why did you remove the php stuff from the readme?

Reply:
I moved it to Branches section